webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -126,6 +129,15 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
         }
     }, [actions, draftStorageId, editingPost.postId]);
 
+    const editedTextHasMentions = () => {
+        const editedMentions = allAtMentions(editText);
+        const originalMentions = allAtMentions(originalText.current);
+
+        const doesAlreadyContainMentions = originalMentions.length > 0;
+
+        return doesAlreadyContainMentions && editedMentions.length > originalMentions.length;
```

Comment:
This condition works but we could get into rare edge cases of maybe updating a letter of a mention because of a typo and the length being the same so the help text won't show?

I believe when we discussed this feature we agreed that to make it simple if there is any mentions in the edit text (regardless if added or previous) then it should show the help text. Can we simplify this condition to behave this way?